docs/Project.toml

```diff
@@ -40,6 +40,7 @@ SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 SciMLSensitivity = "1ed8b502-d754-442c-8d5d-10ac956f44a1"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
```

Comment:
A bit annoying to add a dependency for a tutorial. However, the `density` function feels quite important for stochastic -related analysis, so probably something users should know about and which we might want to use in other tutorials as well. Also, it is part of the Plots system, which should make it less of an addition.

Reply:
We probably should at some point just switch everything to Makie-verse given we are now pulling it in, but that is a longer-term goal I guess.

Reply:
agreed